yarn-project/p2p-bootstrap/src/index.ts

```diff
@@ -18,7 +19,11 @@ async function main(
   telemetryClient: TelemetryClient = new NoopTelemetryClient(),
   logger = debugLogger,
 ) {
-  const bootstrapNode = new BootstrapNode(telemetryClient, logger);
+  // If a data directory is provided in config, then create a persistent store.
+  // Otherwise, create a temporary store.
+  const store = config.dataDirectory ? await createStore('p2p-bootstrap', config, logger) : openTmpStore();
```

Comment:
I think createStore already handles this doesn't it?

Reply:
yes you are correct it does, will update